src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -79,7 +79,7 @@ internal PublicClientApplicationBuilder WithUserTokenLegacyCachePersistenceForTe
         /// </summary>
         /// <param name="enableBroker"></param>
         /// <returns></returns>
-        internal PublicClientApplicationBuilder WithBroker(bool enableBroker)
+        public PublicClientApplicationBuilder WithBroker(bool enableBroker)
         {
             // TODO: * This should become public only on mobile platforms that support using a broker
```

Comment:
This TODO isn't realized - this is now public on all platforms. Consider making it public on iOS and Netstandard. We should have a discussion about the behavior of this method on other platforms, generally the approach can be: 

1. hide on non-supporting platforms at build time
2. do nothing if set (alternatively we can throw)

Reply:
Previously we agreed builders would not throw during construction (perhaps time to re-consider?)...however it should somehow fail fast at run-time. 
IMO: it makes sense to only show this on .netstandard, ios and android

---
In reply to: [254633239](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/874#discussion_r254633239) [](ancestors = 254633239)